Formula/vlang.rb

```diff
@@ -0,0 +1,106 @@
+class Vlang < Formula
+  desc "V programming language"
+  homepage "https://vlang.io"
+  url "https://github.com/vlang/v/archive/0.1.18.tar.gz"
+  sha256 "3f3407a78aca7fc3b42a3fc1f1d2b9724c1e4c71fbd5d37ff12976cd2305cec1"
+
+  def install
+    system "make"
+  end
+
+  test do
+    # test version CLI command
+    version_output = shell_output("#{bin}/v -v")
+    assert_match version.to_s, version_output
+
+    # test help CLI command
+    help_output = shell_output("#{bin}/v -h")
+    assert_match "Usage: v [options] [file | directory]", help_output
```

Comment:
Here, in my opinion only this first assert_match is enough for `v -h`.
The help lines below will be changed many times, which will cause false positives for the build process.

Reply:
We can just remove this test since @fxcoudert said, it's not required.